static/elements/chromedash-roadmap-milestone-card.js

```diff
@@ -1,6 +1,6 @@
 import {LitElement, html, nothing} from 'lit';
 import {ROADMAP_MILESTONE_CARD_CSS} from
-  '../sass/elements/chromedash-roadmap-milestone-card-css.js';
+'../sass/elements/chromedash-roadmap-milestone-card-css.js';
```

Comment:
This auto-formatting continues to happen to me as well. Maybe it's fine to just keep this change?

Reply:
Yeah, I'll merge it and then it should not cause diffs in the future.